src/no/runsafe/runsafeinventories/RegionInventoryHandler.java

```diff
@@ -153,6 +162,7 @@ public Boolean removeInventoryRegion(String worldName, String regionName)
 		// World and region passed all checks, remove that region's inventory.
 		inventoryRegions.remove(worldName, regionName);
 		inventoryRegionRepository.removeInventoryRegion(worldName, regionName);
+		inventoryRepository.wipeInventories(server.getWorld(worldName).getUniverse().getName() + "-" + regionName);
```

Comment:
string concatenation feels wrong here, maybe do it same way the other two methods work?